Move `old-ui-src` files to `client/src` directory

jiexiangfan opened the pull request:
#  Description
Moving `old-ui-src` files to `client/src` directory. Upgrading a few packages and removing/changing attributes according to warnings.
# Trello Ticket
https://trello.com/c/oZSXbWpe
# Type of Change
- [x] Refactoring
# Risk
- React-hook-form is not up to date. 
Having troubles migrating from v6 to v7. Will do it on a separate ticket.
- Header of the landing page is out of position. 
The menu icon is not showing properly.  The collapsable navbar is not working properly. Suspect is because of the Reactstrap package. Thinking of fixing it on a separate ticket.

Commit:
install material ui and react router dom

Commit:
App moved to new files

- can't navigate between page
- errors occured

Commit:
moved assets to new position

Commit:
actions files to new location

Commit:
corrected path of imports

Commit:
moved reducer location

Commit:
moved pages

Commit:
moved everything and changed all imports

Commit:
fix warning, refer to this when needed

riordanalfredo commented:
Hello!

I found out that the UI of the landing page is broken. I reckon it cannot retrieve the CSS files appropriately. Or maybe, because it mixes reactstrap and material UI. Could you please investigate it again? At minimum, we need to find what's the cause of the issue. Thanks! 
![image](https://user-images.githubusercontent.com/17421174/149241509-f8beabfb-fe23-491f-9fd9-3eb8eeb9f017.png)

![image](https://user-images.githubusercontent.com/17421174/149241744-c29ba61e-854b-4fb2-a3c5-6f2604b11108.png)
^Weird margin at the body.

riordanalfredo reviewed (changes requested):
See my comments above

Commit:
changed favicon to deltacore

Commit:
removed redux logo + added mui base css

Commit:
fixed children width larger than parent

with default css, this component width is longer than parents which will cause website scrolling horizontally

Commit:
import bootstrap css to fix landing page header

Commit:
update mr-auto to me-auto

bootstrap 5 update

Commit:
remove style={{width 100%}} to avoid button sticking together

vgoh1 reviewed (approved):
Honestly, the homepage is a bit cooked (mobile view looks veeeerrry off) but I think it can be worked on after the project structure goes in, along with everything else so approving for now
